README.md

```diff
@@ -32,3 +22,21 @@ yarn test -- --coverage
 ```
 When you run tests with coverage, the `coverage/` directory will be created at the root. You can
 open the web version of the coverage report `coverage/lcov-report/index.html` to see which lines are covered by tests.
+
+## Ruling
+
+The ruling test is a special integration test which launches the analysis of a large code base, 
+and then compares those results to the set of expected issues (stored as snapshot files). 
+To have this code base locally:
+```
+ git submodule init
+ git submodule update
+```
+
+To run the ruling test:
+```
+yarn ruling
+yarn ruling --rule <RuleFileName> # to run ruling for a single rule
+yarn ruling --update # to update the snapshots
```

Comment:
I think this should be yarn ruling -- --update (or at least, that way it works, while the other way it doesn't seem to)